kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/AdaptiveBigQueryWriter.java

```diff
@@ -76,10 +80,15 @@ public void performWriteRequest(PartitionedTableId tableId,
     // BigQuery schema updates taking up to two minutes to take effect
     if (writeResponse.hasErrors()
         && onlyContainsInvalidSchemaErrors(writeResponse.insertErrors())) {
-      schemaManager.updateSchema(tableId.getBaseTableId(), topic);
+      try {
+        schemaManager.updateSchema(tableId.getBaseTableId(), topic);
```

Comment:
Previously, this error would percolate up and be caught by the batch size retry logic, which looks for a 400 with an "invalid" message.  If you try to update a table with an invalid schema, BigQuery will return a 400 "invalid". So, without this, there is some weirdness and a lack of clarity whenever this error pops up. This prevents that.